backend/app/schemas/profile.py

```
@@ -41,10 +42,28 @@ def validate_passout_year(cls, value: int | None) -> int | None:
 
 
+
 class ProfileCreate(ProfileBase):
     educations: list[EducationCreate] = Field(default_factory=list)
     work_experiences: list[WorkExperienceCreate] = Field(default_factory=list)
 
+    @field_validator("resume_url")
+    @classmethod
+    def validate_resume_url(cls, value: HttpUrl | None) -> HttpUrl | None:
+        if value is not None:
+            if "drive.google.com" not in str(value):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)